### PR TITLE
Mu0n wave 9 ships

### PR DESCRIPTION
**Pull request by Mu0n**
or is it wave 10? anyway, september 2021 stuff.

### CODE REVIEW

File: data/upgrades/astromech.json [hunk 1]
```diff
@@ -559,5 +559,53 @@
...
     "restrictions": [{ "factions": ["Scum and Villainy"] }]
+  },
+  {
+    "name": "Watchful Astromech",
+    "limited": 0,
+    "xws": "watchfulastromech",
+    "hyperspace": true,
+    "sides": [
+      {
+        "ability": "After you perform a [Reload] or [Rotate Arc] action, i you are in an enemy ship's firing arc, you may perform a red [Calculate]",
```
Comment (danrs) [inline data/upgrades/astromech.json:570]: Typo in ability, should be "After you perform a [Reload] or [Rotate Arc] action, if you are in an enemy ship's firing arc, you may perform a red [Calculate] action."

File: data/upgrades/astromech.json [hunk 2]
```diff
@@ -559,5 +559,53 @@
...
+        "type": "Astromech",
+        "slots": ["Astromech"],
+        "ffg": -999,
+        "artwork": "????",
+        "grants": [
+          {
+            "type": "action",
+            "value": {
+              "type": "Rotate Arc",
+              "linked": { "type": "Coordinate", "difficulty": "White" }
```
Comment (danrs) [inline data/upgrades/astromech.json:598]: Links to calculate, not coordinate (now wouldn't that be something!)

File: data/pilots/first-order/tie-wi-whisper-modified-interceptor.json [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+      "initiative": 5,
+      "limited": 1,
+      "shipAbility": {
+        "name": "Heavy Weapon Turret",
+        "text": "You can rotate your [Single Turret Arc] indicator only to your [Front Arc] or [Rear Arc]. You must treat the [Front Arc] requirement of your equipped [Missile] upgrades as [Single Turret Arc]."
+      },
+      "hyperspace": true,
+      "cost": 0,
+      "slots": [
+        "Talent",
```
Comment (danrs) [inline data/pilots/first-order/tie-wi-whisper-modified-interceptor.json:67]: I'm not sure if we know the slots for all the pilots yet. Presumably Kylo has one talent and one force? Probably better to leave slots out entirely until release for the pilots we don't know

File: data/pilots/first-order/tie-wi-whisper-modified-interceptor.json [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+    {
+      "name": "709th Legion Ace",
+      "xws": "709thlegionace",
+      "initiative": 4,
+      "limited": 0,
+      "shipAbility": {
+        "name": "Heavy Weapon Turret",
+        "text": "You can rotate your [Single Turret Arc] indicator only to your [Front Arc] or [Rear Arc]. You must treat the [Front Arc] requirement of your equipped [Missile] upgrades as [Single Turret Arc]."
+      },
+      "text": "Commanded by Supreme Leader Kylo Ren, the 709th Legion fights without mercy or remorse to crust the Resistance and bring the galaxy under the Forst Order's dominion.",
```
Comment (danrs) [inline data/pilots/first-order/tie-wi-whisper-modified-interceptor.json:115]: Text should be: "Commanded by Supreme Leader Kylo Ren, the 709th Legion fights without mercy or remorse to crush the Resistance and bring the galaxy under the First Order's dominion."

File: data/pilots/first-order/tie-wi-whisper-modified-interceptor.json [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+      "hyperspace": true,
+      "cost": 0,
+      "slots": [
+        "Talent",
+        "Talent",
+        "Missile",
+        "Tech",
+        "Configuration"
+      ],
+      "ability": "After you perform a [Bullseye Arc] attack, if you have 1 or more non-lock red or orange tokens, you may perform a bonus attack against a different target.",
```
Comment (danrs) [inline data/pilots/first-order/tie-wi-whisper-modified-interceptor.json:125]: copy paste error, no ability on 709th legion ace

File: data/pilots/first-order/tie-wi-whisper-modified-interceptor.json [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+      "ffg": -999,
+      "artwork": "???",
+      "image": "???",
+      "caption": "Herald of Destruction",
```
Comment (danrs) [inline data/pilots/first-order/tie-wi-whisper-modified-interceptor.json:129]: copy paste error, no caption on 709th ace

File: data/pilots/first-order/tie-se-bomber.json [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+      "slots": [
+        "Talent",
+        "Tech",
+        "Torpedo",
+        "Missile",
+        "Gunner",
+        "Device",
+        "Modification"
+      ],
+      "text": "The live-fire exercices of the First Order are intended to root out the weak, enforcing the maxim that only the strong survive.",
```
Comment (danrs) [inline data/pilots/first-order/tie-se-bomber.json:151]: Text should be: "The live-fire exercises of the First Order are intended to root out the weak, enforcing the maxim that only the strong survive."

**Review by danrs — CHANGES_REQUESTED** [review]
A few typos, and I think we should consider removing slots on pilots we don't know yet (or else we just have to update them pronto once we know actual slots)

**Review by Mu0n — COMMENTED** [review]
all good

**Pull request merged**